packages/flutter_tools/lib/src/device.dart

```diff
@@ -1159,6 +1159,7 @@ class DebuggingOptions {
     Map<String, Object?> platformArgs, {
     bool ipv6 = false,
     DeviceConnectionInterface interfaceType = DeviceConnectionInterface.attached,
+    bool isCoreDevice = false,
```

Comment:
may be helpful to document what core device is here (i know you have it in core_devices)

Reply:
I added a comment where it's used in this function